src/IECoreMaya/SceneShapeInterface.cpp

```diff
@@ -1470,3 +1484,81 @@ SceneInterface::Path SceneShapeInterface::fullPathName( std::string relativeName
 	return fullPath;
 }
 
+namespace
+{
+	bool parseConvertParams( std::map<std::string, std::string> &params, const MString &paramsStr )
```

Comment:
No, we would have to call into python, which could be a downside potentially. But its just at conversion time, not at runtime, so maybe it'd be alright.